cns/fsnotify/fsnotify.go

```diff
@@ -5,18 +5,25 @@ import (
 	"io"
 	"io/fs"
 	"os"
+	"runtime"
 	"sync"
 	"time"
 
 	"github.com/Azure/azure-container-networking/cns"
+	"github.com/Azure/azure-container-networking/cns/configuration"
+	"github.com/Azure/azure-container-networking/cns/hnsclient"
+	"github.com/Azure/azure-container-networking/cns/restserver"
 	"github.com/fsnotify/fsnotify"
 	"github.com/pkg/errors"
 	"go.uber.org/zap"
 	"golang.org/x/sync/errgroup"
 )
 
+const DefaultNetworkID = "azure"
+
 type releaseIPsClient interface {
 	ReleaseIPs(ctx context.Context, ipconfig cns.IPConfigsRequest) error
+	GetEndpoint(ctx context.Context, endpointID string) (*restserver.GetEndpointResponse, error)
```

Comment:
should we change interface name or add to new interface?

Reply:
I can create a new interface for this.

Reply:
changed it to IPAMCLient

Reply:
How about `endpointManager`?